docs/API.md

```diff
@@ -801,18 +799,19 @@ In the case of a test created with `start_domain_test`:
 * `"creation_time"`: A *timestamp*. The time at which the *test* was enqueued.
 * `"id"`: An integer.
 * `"hash_id"`: A *test id*. The *test* in question. 
-* `"params"`: The `"params"` object sent to `start_domain_test` when the *test*
-  was started.
+* `"params"`: A normalized `"params"` object sent to `start_domain_test` when
+  the *test* was started.
 * `"results"`: A list of *test result* objects.
 
 
 In the case of a test created with `add_batch_job`:
 * `"creation_time"`: A *timestamp*. The time at which the *test* was enqueued.
 * `"id"`: An integer.
 * `"hash_id"`: A *test id*. The *test* in question. 
-* `"params"`: The `"params"` object sent to `start_domain_test` when the *test*
-  was started.
-* `"results"`: the result is a list of *test id* corresponding to each tested domain.
+* `"params"`: A normalized `"params"` object sent to `start_domain_test` when
+  the *test* was started.
```

Comment:
It should be `add_batch_job`, shouldn't it?

```
A normalized version of the `"params"` object sent to `add_batch_job` when
  the *test* was started.
```

Reply:
Yes, fixed in 558f9c6